src/main/java/mServer/crawler/sender/phoenix/tasks/PhoenixOverviewTask.java

```diff
@@ -0,0 +1,76 @@
+package mServer.crawler.sender.phoenix.tasks;
+
+import com.google.gson.reflect.TypeToken;
+import de.mediathekview.mlib.Config;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'de.mediathekview.mlib.Config'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)